docs/manual.txt

```diff
@@ -956,6 +956,20 @@ In that case, the template engine used will be your theme's and the arguments yo
 as well as the global context from your ``conf.py``, are available to the template you
 are creating.
 
+.. sidebar:: The Global Context
```

Comment:
Should be a full section. We should also have some information about `.. data:` in a more discoverable place than the large „other metadata fields” sidebar.